### ISSUE TITLE
False W002 error

### ISSUE THREAD

**Issue by JeyKeu** [twbs/bootlint#230]
This works:

```html
<meta http-equiv="X-UA-Compatible" content="IE=edge />
```
Whereas this does not:

```html
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
```

`,chrome=1` causes bootlint to complain.

**Comment by cvrebert** [comment]
Duplicate of #152.
